remotecv/__init__.py

```diff
@@ -1 +1 @@
-__version__ = "3.2.2"
+__version__ = "3.3.2"
```

Comment:
Hey @guilhermef, maybe we can reset the patch version, since you increment minor version.
What do you think?!

Reply:
Good catch, thanks @RaphaelVRossi